lib/client.js

```diff
@@ -11,6 +11,17 @@ const DEFAULT_PORT = 8443;
 const DEFAULT_VERIFY_CERT = true;
 const DEFAULT_VERSION = 1;
 
+const MODULE_VERSION = `${
+  require('../package.json').name
```

Comment:
package name and version should be fetched from process.env instead of using require:

- process.env.npm_package_name
- process.env.npm_package_version

Reply:
I checked that this does not work in all cases. if we run the script through `node index.js`, then `npm` environment variables are not loaded. Then we get `undefined@undefined`.

Reply:
discussed.  let's use package.json to define the $user_agent and process.env to define the $user_application (if it exists)